.chloggen/limit-volume-inst.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: bug_fix
```

Comment:
I think we should consider this an enhancement :)

Reply:
+1 changed